android/app/src/main/java/com/google/codeu/chatme/model/Conversation.java

```diff
@@ -51,6 +52,15 @@ public void setTimeCreated(long timeCreated) {
         this.timeCreated = timeCreated;
     }
 
+    public String getLastMessageContent() {
+        return this.lastMessage.getContent();
+    }
+
+    public String getReadableLastMessageTime() {
+        // TODO: format string
```

Comment:
looks like some freaky code language or something with time represented in seconds lol